tests/sdk/test_post_markdown.py

```diff
@@ -0,0 +1,175 @@
+from sdk.post_markdown import PostMarkdown
+import pytest
+
+
+MD = '''
+# HEADER
+
+SOME TEXT 1
+
+!skip
+```python
+a = 1
+```
+
+!continue
+```python
+a += 1
+```
+
+!skip, continue
+```python
+assert a == 2
+```
+
+SOME TEXT 2
+'''
```

Comment:
I find the code much easier to read when all lines that belong to the same expression are indented:

```python
MD_TELEGRAM = dedent("""
    # HEADER
    ...
""")
```